frontend/src/metabase/query_builder/selectors.js

```diff
@@ -459,6 +471,35 @@ export const getIsResultDirty = createSelector(
   },
 );
 
+export const getIsResultDirtyIgnoringModels = createSelector(
```

Comment:
This is basically `getIsResultDirtyIgnoringModels`, but in the end we don't call `areModelsEquivalent`.